substrate/frame/balances/src/lib.rs

```diff
@@ -767,6 +767,29 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Burn the specified liquid free balance from the origin account.
+		///
+		/// If the origin's account ends up below the existential deposit as a result
```

Comment:
Can you maybe quickly explain how this is different from sending it to the `0x0..` account? (mainly TotalIssuance adjustment i guess)

Reply:
Not sure I get the question - the functionality is very similar to what Ethereum has, only with added `keep_alive` flag since for us the accounts can be reaped.

Am I misunderstanding the question? 😅

Reply:
>the functionality is very similar to what Ethereum has

No its different. When you send funds to `0x0...`, then the Total Issuance stays the same.  
But by burning funds, you reduce the Total Issuance and make the existing amounts be a larger fraction of the total.

So to say: Ethereum does not have proper burning - only means of making funds permanently inaccessible.

Reply:
Oh, than I thought wrong 🤔. I thought that the _burn logic_ of the `0x00...00` address is part of the state transition function. I've researched a bit more now, and see that's not the case 🙈 

I will add a comment to make this more clear as you suggested.